CHANGELOG.md

```diff
@@ -2,6 +2,9 @@
 
 ## Next Release
 
+#### Breaking Changes
```

Comment:
I think it not a breaking change. as we are preparing django-auditlog for major release 3 and it's already in beta. 
```suggestion
#### Improvements
```